src/main/resources/institutional-config.properties

```diff
@@ -47,7 +47,6 @@ cu.spring.source.files=\
   classpath:edu/cornell/kfs/module/cg/cu-spring-cg.xml,\
   classpath:edu/cornell/kfs/gl/cu-spring-gl.xml,\
   classpath:edu/cornell/kfs/fp/cu-spring-fp.xml,\
-  classpath:edu/cornell/kfs/sec/cu-spring-sec.xml,\
```

Comment:
If CuSecurityModelMaintainableImpl is being kept, then this "cu-spring-sec" entry should be kept in place.

Reply:
fixed